testing/rhythmbox/APKBUILD

```diff
@@ -11,7 +11,9 @@ depends_dev="gnome-desktop-dev gstreamer-dev libpeas-dev"
 makedepends="$depends_dev tdb-dev libsoup-dev json-glib-dev totem-pl-parser-dev gmime-dev libmtp-dev gst-plugins-base-dev py-gobject3-dev libxslt libxml2-utils docbook-xml docbook-xsl itstool intltool"
 install=""
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://download.gnome.org/sources/rhythmbox/3.4/rhythmbox-$pkgver.tar.xz"
+source="https://download.gnome.org/sources/rhythmbox/3.4/rhythmbox-$pkgver.tar.xz
+	fix-preprocess-token.patch	
```

Comment:
There is whitespace before EOL here. I'll fix that up for you.

Reply:
Thanks ncopa!